tensorflow/c/experimental/gradients/grad_test_helper.cc

```diff
@@ -105,6 +106,36 @@ void CheckTensorValue(AbstractTensorHandle* t, absl::Span<const float> manuals,
   delete[] danalytical;
 }
 
+Model BuildGradModel(Model ops, size_t num_inputs, string ops_name,
```

Comment:
Do we need to provide `num_inputs` here or could we use `inputs.size()` ?

Reply:
Done